src/reactor.h

```diff
@@ -299,6 +300,11 @@ class Reactor : public cyclus::Facility {
   }
   double power_cap;
 
+  #pragma cyclus var { \
+    "default": "power", \
+    "doc": "The name of the 'power' commodity used in conjunction with a deployment curve.", \
+  }
+  std::string power_commod;
```

Comment:
Not blocking, but it would be nice to have the new power vars moved above the hidden variables for readability.